pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -525,3 +529,53 @@ def test_all(self):
             RPM_UNSIGNED_FEED_COUNT,
             repo['content_unit_counts']['rpm'],
         )
+
+
+class SyncZchunkRepoSkipTestCase(unittest.TestCase):
+    """Sync feed with ``zchunks`` and ensure no ``zchunk`` units exist.
+
+    A new compression type for repodata exists in Fedora 30 called
+    ``zchunks``. These can be created with patches made to ``createrepo_c --zck``.
+
+    Pulp 2 will ignore this repodata archive type to preserve the quality of the
+    existing data. At the time of writing this test, ``zchunk`` data is
+    in addition to standard repodata. There are currently no cases of
+    repodata only containing ``zchunk`` archives.
+
+    This test targets the following issues:
+
+    `Pulp #4530 <https://pulp.plan.io/issues/4529>`_
+    `Pulp #4530 <https://pulp.plan.io/issues/4530>`_
+
+    Steps:
+
+    1. Clean all existing orphans to prevent false failures.
+    2. Create a repo point to a ``zchunk`` feed and sync it.
+    3. Assert no local units with the ``.zck`` extension.
+
+    """
+
+    def test_zchunk_sync(self):
+        """Sync a ``zchunk`` repo and verify Pulp 2 does no sync ``.zck`` data."""
+        cfg = config.get_config()
+        if check_issue_4529(cfg):
+            self.skipTest('https://pulp.plan.io/issues/4529')
+        client = api.Client(cfg, api.json_handler)
+        client.delete(ORPHANS_PATH)
```

Comment:
What failures did you face without cleaning orphans?

Reply:
Perhaps, later we can explore this a bit more.
 
1. Clean orphans as part of a SetUpClass
2. Sync the zchunck more than once, and assure that there are no failures.

If I understood the failure properly.

Reply:
Since this is a unit type that we are not expecting and we are searching the file system, it makes sense to clean the orphans on the system before running the `find` on the content units.  

We could NOT run the clean, but if there was anything there with a `.zck` extension, the test would vacuously fail.

In cases were the operating system is being searched manually, I believe it is better to take the short amount of time to clean rather than have false failures.